redis/client.py

```diff
@@ -561,7 +561,7 @@ class Redis:
     """
     RESPONSE_CALLBACKS = {
         **string_keys_to_dict(
-            'AUTH EXPIRE EXPIREAT HEXISTS HMSET MOVE MSETNX PERSIST '
+            'AUTH EXPIRE EXPIREAT HEXISTS HMSET LOVE MOVE MSETNX PERSIST '
```

Comment:
Is this supposed to be LMOVE? Should we also have BLMOVE here?

Reply:
Yes and updated. More interesting that it didn't fail the tests. Probably should run vulture against these.